index.html

```diff
@@ -4240,47 +4240,67 @@ <h2>HTTP</h2>
+and shape that their application already understands such that they do not need to transform it in
+order to consume its contents. Notably, this does not mean that consumers do not need to understand any context at all, rather, consuming applications only need to understand a chosen set of contexts and document shapes to work with and not others. Issuers can publish contexts and information about their <a>verifiable credentials</a> to aid consumers who do not use generalized processors, just like would be done with any other JSON-formatted data.
+          </p>
+          <p>
+Processing <a>verifiable credentials</a> or <a>verifiable presentations</a>
+by transforming JSON-LD to RDF might be desirable under the following conditions:
           </p>
-
           <ul>
             <li>
-Before securing or after verifying content
-that requires <a href="https://csrc.nist.gov/glossary/term/data_integrity">data
-integrity</a>, such as a
-<a>verifiable credential</a> or <a>verifiable presentation</a>.
+Before securing or after verifying <a>verifiable credentials</a> or <a>verifiable presentations</a>,
+in cases where the issuer made use of the underlying RDF Dataset model to provide details regarding 
+their intended expression of the <a href="#verifiable-credential-graphs">Verifiable Credential Graphs</a>.
             </li>
             <li>
-When performing JSON Schema validation, as described in Section
+After performing JSON Schema validation, as described in Section
 <a href="#data-schemas"></a>.
             </li>
             <li>
 When serializing or deserializing <a>verifiable credentials</a> or
-<a>verifiable presentations</a> into systems that store or index their contents.
+<a>verifiable presentations</a> into systems that store or index data in 
+Turtle [[?TURTLE]], N-Quads [[?N-QUADS]], or JSON-LD [[?JSON-LD11]].
             </li>
             <li>
-When operating on <a>verifiable credentials</a> or <a>verifiable
-presentations</a> in a software application, after verification or validation
-is performed for securing mechanisms that require an understanding of
-and/or processing of JSON-LD.
+When consuming <a>verifiable credentials</a> or <a>verifiable
+presentations</a> in a software application, after verification 
+is performed for securing mechanisms that do not require JSON-LD transformations.
             </li>
             <li>
-When an application chooses to process the media type using the `+json`
+When an application chooses to process the media type using the `+ld+json`
 structured media type suffix.
             </li>
           </ul>
 
           <p>
-That is, JSON processing is allowed as long as the document being consumed or
-produced is a <a>conforming document</a>. If JSON processing is desired, an
-implementer is advised to follow the following rule:
+Instead of performing general JSON-LD processing, application-specific processing is allowed as long as the document being consumed or
```

Comment:
just a line break
```suggestion
Instead of performing general JSON-LD processing, application-specific
processing is allowed as long as the document being consumed or
```